ui/pages/settings/advanced-tab/advanced-tab.container.js

```diff
@@ -75,6 +80,8 @@ export const mapDispatchToProps = (dispatch) => {
     setAdvancedInlineGasFeatureFlag: (shouldShow) =>
       dispatch(setFeatureFlag('advancedInlineGas', shouldShow)),
     setUseNonceField: (value) => dispatch(setUseNonceField(value)),
+    setSpeedUpAnyTransaction: (shouldShow) =>
+      dispatch(setFeatureFlag('speedUpAnyTransaction', shouldShow)),
```

Comment:
Instead of modifying this, we can pull the state into the hook itself so this file should end up being unchanged.